wiki/People/Performance_Points_Committee/en.md

```diff
@@ -9,7 +9,9 @@ The **Performance Points Committee** is a panel of well-versed players and devel
 
 The osu! panel was first announced to the public through the [November 2021 *Performance Points & Star Rating Updates* news post](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates) in response to a [community survey](https://osu.ppy.sh/home/news/2021-08-17-pp-sr-survey) held earlier that year.
 
-osu!mania's panel was formed 22 April 2022 in anticipation of a surge in active development effort, starting with [a change in difficulty calculation for hold note releases in osu!mania](https://github.com/ppy/osu/pull/17913).
+The osu!mania panel was formed 22 April 2022 in anticipation of a surge in active development effort, starting with [a change in difficulty calculation for hold note releases in osu!mania](https://github.com/ppy/osu/pull/17913).
+
+The osu!taiko panel was formed on 1 May 2022 for reasons similar to the above, with the first batch of changes expected to roll out shortly.
```

Comment:
The reason this is in a comment is I have two recommendations:

The osu!taiko panel was formed on 1 May 2022 in response to a [community survey](https://osu.ppy.sh/home/news/2022-01-14-taiko-mania-pp-sr-survey) held earlier that year, starting with a [document](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit?usp=sharing) outlining the information of the changes to difficulty calculation.

Or, if a outside document cannot be linked within the wiki then:

The osu!taiko panel was formed on 1 May 2022 in response to a [community survey](https://osu.ppy.sh/home/news/2022-01-14-taiko-mania-pp-sr-survey) held earlier that year.

Reply:
second option sounds fine